logpolicy/logpolicy.go

```diff
@@ -450,14 +451,14 @@ func tryFixLogStateLocation(dir, cmdname string) {
 
 // New returns a new log policy (a logger and its instance ID) for a
 // given collection name.
-func New(collection string) *Policy {
-	return NewWithConfigPath(collection, "", "")
+func New(collection string, netMon *netmon.Monitor) *Policy {
```

Comment:
can it be nil?

if the answer is always yes, then maybe the type itself needs docs saying a nil Monitor is always valid (not sure whether that's true) and what that means.